### ISSUE TITLE
Explanation does not match code example in User Guide / Expressions / Aggregation

### ISSUE THREAD

**Issue by w00kie** [pola-rs/polars#17858]
### Description

In the `docs/user-guide/expressions/aggregation.md` page, the explanation for Basic Aggregations does not match the example code.

<img width="783" alt="image" src="https://github.com/user-attachments/assets/d9115b73-8cf1-488e-b5e4-0deba3ae96a6">

### Link

https://docs.pola.rs/user-guide/expressions/aggregation/

**Comment by lukemanley** [comment]
I think this can be closed as the user guide has since been updated: 

https://docs.pola.rs/user-guide/expressions/aggregation/#basic-aggregations

**Comment by uurl** [comment]
I checked this against current `main`, and the aggregation guide text now appears to match the referenced Python example:

- groups by `first_name`
- uses `pl.len()` to count rows per group
- collects `gender` values into a list via `pl.col("gender")`
- uses `pl.first("last_name")`

So this may have been fixed incidentally since the issue was opened.

Unless I am missing another mismatch, I think this issue can be closed.

**Comment by MarcoGorelli** [comment]
yeah i think https://github.com/pola-rs/polars/pull/18003 was meant to close this
